gratipay/billing/payday.py

```diff
@@ -65,7 +65,7 @@ class Payday(object):
     want to use their balance at the start of Payday. Balance changes should be
     atomic globally per-Payday.
 
-    Here's the call structure of the Payday.run method:
+    Here's the call structure of the Payday.run method::
```

Comment:
Yeah, it's necessary for code blocks. If I remove it and `make doc` I get an error:

```
Warning, treated as error:
/Users/whit537/personal/gratipay/gratipay.com/gratipay/billing/payday.py:docstring of gratipay.billing.payday.Payday:20: WARNING: Definition list ends without a blank line; unexpected unindent.
```

Here's the [docs](http://www.sphinx-doc.org/en/1.4/rest.html#source-code):

> Literal code blocks ([ref](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks)) are introduced by ending a paragraph with the special marker `::`.